practice/src/css/sections/package.css

```diff
@@ -115,4 +113,12 @@
   .package-card {
     padding-left: 78px;
   }
+
+  .btn-package {
+    padding: 14px 24px;
```

Comment:
I see that you use padding 14px 24px for many button in your site, not only btn package. You should create a common class here and move it to button.css